go/tools/builders/compile.go

```diff
@@ -158,16 +158,15 @@ func compile(args []string) error {
 	// Run nogo concurrently.
 	var nogoOutput bytes.Buffer
 	nogoFailed := false
+	factOut := strings.TrimSuffix(*outExport, filepath.Ext(*outExport)) + ".fact"
```

Comment:
Rename `factOut` to `outFact` for consistency with `outExport`.

Also, let's give the file a constant name so we can easily extract it from an archive. I don't think we'll need to store multiple facts files in an archive. How about `nogo.out`?

For reference, the `go` command uses the name `vet.out`, but it's stored in the build cache as a separate file, not appended to the archive.